composer.json

```diff
@@ -16,7 +16,7 @@
         "padraic/humbug_get_contents": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~6.0"
```

Comment:
This means that you cannot test the library with any PHP version before 7.0. Doesn't it make sense to drop support for these PHP version then?